templates/index.html

```diff
@@ -5,8 +5,9 @@
     <div class="container">
       <div class="row mt-3">
         <div class="col-md-4 mx-auto">
+		  <center><span><i><h4>All fields marked with * are mandatory</h4></i></span></center><br>
           <div class="form-group">
-            <label class="heading" id="label" for="email">Email</label>
+            <label class="heading" id="label" for="email">* Email</label>
```

Comment:
Please make it  "Email *" instead of "* Email"

Reply:
I'm sorry, I don't think I got it exactly.
Do you want me to make it `Email *` instead of `* Email `?

Reply:
Yes,  because it destroys the layout
Image you have a list

1. item
2. more stuff
3. simpy the best

and then have it
1. item
2. * more stuff
3. simpy the best
it destroys the layout. As opposed to an annotation
1. item
2. more stuff *
3. simpy the best

Reply:
True that! Will get that rectified!